### PR TITLE
docs: syntax highlighting

### PR DESCRIPTION
**Pull request by Karitham**
fixes syntax highlighting in docs as well as gender neutral pronouns

### CODE REVIEW

File: docs/hydra/guides/login.mdx [hunk 1]
```diff
@@ -131,7 +131,7 @@ Check out our [reference implementation](https://github.com/ory/hydra-login-cons
 :::
 
 ```typescript title="routes/login.ts"
-// This is the endpoint the user ends up at once she/he inserts their password and username and hits "Log in".
+// This is the endpoint the user ends up at once they insert their password and username and hits "Log in".
```
Comment (kevgo) [inline docs/hydra/guides/login.mdx:134]: Great idea to remove the gender here, let's pluralize the entire sentence.

```suggestion
// This is the endpoint the users end up at once they insert their password and username and hits "Log in".
```

**Review by kevgo — COMMENTED** [review]
Thanks so much for these fixes! One tiny suggestion and then this is ready to :ship:

File: docs/hydra/guides/login.mdx [hunk 2]
```diff
@@ -131,7 +131,7 @@ Check out our [reference implementation](https://github.com/ory/hydra-login-cons
 :::
 
 ```typescript title="routes/login.ts"
-// This is the endpoint the user ends up at once she/he inserts their password and username and hits "Log in".
+// This is the endpoint the users end up at once they insert their password and username and hits "Log in".
```
Comment (vinckr) [inline docs/hydra/guides/login.mdx:134]:
```suggestion
// This is the endpoint the users end up at once they insert their password and username and hit "Log in".
```

missed one ;-)

**Review by vinckr — APPROVED** [review]
lgtm, thanks 🙌

**Review by kevgo — APPROVED** [review]

**Pull request merged**